### PR TITLE
fix(learn): typo in an answer to the Numpy question

### PR DESCRIPTION
**Pull request by pkarczma**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

I found a small typo in one of the answers when going through the course so I thought I could contribute a little bit to this awesome project.

### CODE REVIEW

**Comment by nhcarrigan** [comment]
Hello~!

Thank you for your interest in contributing! Please take a moment to review our [contributing guidelines](https://contribute.freecodecamp.org). In general, changes should be made to only one language in a PR - currently your PR modifies both an English and Chinese file. 🙂

**Comment by pkarczma** [comment]
Hi, thanks for the information! I actually looked there and I wasn't sure if I should make a change in the second (Chinese) file as it contains English text as well - I guess the multiple language feature isn't fully rolled out yet, is it?

**Comment by nhcarrigan** [comment]
I'm not 100% familiar with the new translation flow yet, but my understanding is that changes to the English files get rolled out to the rest of the language files as translations are approved.

**Comment by RandellDawson** [comment]
This change is not necessary.  `It's` is perfectly fine.

**Comment by pkarczma** [comment]
I see, so it's a bit weird to me but I reverted the change in the Chinese file according to your preference. I guess this is how it should be done!

**Review by RandellDawson — CHANGES_REQUESTED** [review]
We can just use `It's`.

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/numpy-introduction-a.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ question:
     - |
       Often you won't work directly with Numpy.
     - |
-      It's is extremely slow.
+      It is extremely slow.
```
[the remark this answers is not in the document]
Comment (pkarczma) [reply curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/numpy-introduction-a.md:33]: Sounds good to me!

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@nhcarrigan If you are fine with these changes, approve/merge.

**Comment by nhcarrigan** [comment]
@RandellDawson I'm good with this, but to clarify - you're good with `It is` over `It's`?

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/numpy-introduction-a.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RandellDawson) [reply curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/numpy-introduction-a.md:33]: @pkarczma Can you please click the Commit suggestion button?

**Review by moT01 — APPROVED** [review]
LGTM @pkarczma 🎉 Congratulations on your first contribution to freeCodeCamp 🎉 We welcome future contributions 😄

**Pull request merged**